### PR TITLE
config: support config resolve lock lite threshold

### PR DESCRIPTION
**Pull request by lysu**
ref https://github.com/pingcap/tidb/pull/25835

support change resolve lock lite threshold value in configuration and change nothing else

### CODE REVIEW

**Review by youjiali1995 — APPROVED** [review]

**Review by sticnarf — APPROVED** [review]
LGTM

File: integration_tests/2pc_test.go [hunk 1]
```diff
@@ -1176,6 +1175,7 @@ func (s *testCommitterSuite) TestResolveMixed() {
 	atomic.StoreUint64(&transaction.ManagedLockTTL, 100)        // 100ms
 	defer atomic.StoreUint64(&transaction.ManagedLockTTL, 3000) // restore default value
 	ctx := context.Background()
+	bigTxnThreshold := 16
```
Comment (disksing) [inline integration_tests/2pc_test.go:1178]: you can add a function to `LockResolverProbe` to get the configuration.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**